rdkit/Chem/Pharm3D/UnitTestPharmacophore.py

```diff
@@ -28,8 +25,8 @@ def feq(n1, n2, tol=1e-5):
 class TestCase(unittest.TestCase):
 
   def setUp(self):
-    self.fdefBlock = \
-                   """DefineFeature HAcceptor1 [N,O;H0]
+    self.fdefBlock = """
```

Comment:
same as above: This is now a different string than before, so there should be an actual reason for this change.